src/operator/bufferCount.ts

```diff
@@ -19,7 +19,11 @@ import {Observable} from '../Observable';
  * @returns {Observable<T[]>} an Observable of arrays of buffered values.
  */
 export function bufferCount<T>(bufferSize: number, startBufferEvery: number = null): Observable<T[]> {
-  return this.lift(new BufferCountOperator(bufferSize, startBufferEvery));
+  return this.lift(new BufferCountOperator<T>(bufferSize, startBufferEvery));
+}
+
+export interface BufferCountSignature<T> {
```

Comment:
One way to work around extra import is to use interface merging: `export interface bufferCount`. Now no extra `import` is required for `BufferSignature`, `bufferCount` acts as both function and signature. The benefit is marginal however.